hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/executorservice/ExecutorServiceSubmitToPartitionMessageTask.java

```diff
@@ -87,7 +87,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "submitToPartition";
```

Comment:
The user methods using this message task are submit and submitToKeyOwner. I think we should use one of these. WDYT?

Reply:
See above [comment](https://github.com/hazelcast/hazelcast/pull/25020#discussion_r1265022035).